OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/StaticFragmentPagerAdapter.java

```diff
@@ -25,8 +28,14 @@
     @NonNull
     private final Map<Integer, Fragment> positionToFragment = new HashMap<>();
 
-    public StaticFragmentPagerAdapter(@NonNull FragmentManager manager, @NonNull FragmentItemModel... items) {
+    private FragmentCreated fragmentCreated;
+    public interface FragmentCreated {
+        void onFragmentCreated();
```

Comment:
Its confusing with the onCreate function, `onFragmentInstantiated` will be a good name.